src/PrestaShopBundle/Form/Admin/Sell/Customer/CustomerType.php

```diff
@@ -306,6 +307,20 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     ),
                     'required' => false,
                     'invalid_message' => $this->trans('This field is invalid.', 'Admin.Notifications.Error'),
+                    'constraints' => [
+                        new Range([
+                            'min' => 0,
+                            'max' => 4294967295,
```

Comment:
Can you make it a const somewhere? 🙃

Reply:
Very good catch that leads to further thinking...
1. PHP doesn't use unsigned integers, only signed integers. It means the max integer value should be: 2,147,483,647
2. Integers range depends on the platform: 32 bits (2 billions above) or 64 bits (9E18)
3. There is a native PHP constant, platform dependant, for max value: `PHP_INT_MAX`

Reply:
No apologies : it's a pleasure to have feedback.

@atomiix I have done

@saulaski We are a platform. We should limit technically, not for business point of view. Because relative to the business, we can be lower for a specific business and higher for an another one.